### PR TITLE
Max quiz complete

### PR DESCRIPTION
**Pull request by maxkrieg**

### CODE REVIEW

File: quiz.rb [hunk 1]
```diff
@@ -1,20 +1,68 @@
...
 # Make all of those attributes read-only.
+class Dog
+
+  def initialize(name,age,breed)
+    @name = name
+    @age = age
+    @breed = breed
+  end
+
+  attr_reader :age
```
Comment (GA-MEB) [inline quiz.rb:13]: One tip: you can actually write all of these in a single line, separating each symbol with commas.
e.g.
`attr_reader :name, :breed, :age`
Order doesn't matter.

File: quiz.rb [hunk 2]
```diff
@@ -1,20 +1,68 @@
...
+  attr_reader :name
+  attr_reader :breed
+
+end
+
 
 # Question 2
 # Create a new Dog object.
 
+Spot = Dog.new("spot",7,"black lab")
```
Comment (GA-MEB) [inline quiz.rb:23]: Don't capitalize `Spot` here - reserve capital letters for class names.

File: quiz.rb [hunk 3]
```diff
@@ -1,20 +1,68 @@
...
+  attr_reader :age
+
+end
+
 
 # Question 4
 # Create a new Person object using parameters "Dave", 32, and "Ohio".
 # Then, set that Person's location to "Somerville"
 
+Dave = Person.new("Dave",32,"Ohio")
```
Comment (GA-MEB) [inline quiz.rb:48]: See above.

File: quiz.rb [hunk 4]
```diff
@@ -1,20 +1,68 @@
...
 # Name and location should be writeable, but age should be read-only.
+class Person
+
+  def initialize(name,age,location)
+    @name = name
+    @age = age
+    @location = location
+  end
+
+  attr_accessor :name
```
Comment (GA-MEB) [inline quiz.rb:37]: See above. It's also convention to put `attr_accessor` and `attr_reader` before the `initialize` method.

**Comment by GA-MEB** [comment]
5/5 - nice work! - but watch out for those conventions.

**Comment by maxkrieg** [comment]
Thanks for all the detailed feedback Matt! Super helpful

**Pull request closed without merging**